src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const isNotMarried = isMarried
```

Comment:
when variables name starts from "is...", it means that values have been boolean type.

isNotMarried  = empty string  - it looks a little bit strange and not obvious.

You can name it  like "notMarriedStatusMessage"

But it it will be better to use next logic:

```suggestion
  const partnerType = (sex === 'm')
    ? `wife`
    : `husband`;
    
    and then, in murkup....
    
    <p className="Person__partner">
        {
          isMarried
            ? `${partnerName} is my ${partnerType}`
            : `I am not married`
        }
      </p>
```

Reply:
fixed!